sdk/core/core-lro/review/core-lro.api.md

```diff
@@ -70,6 +92,7 @@ export class PollerCancelledError extends Error {
 
 // @public
 export interface PollerLike<TState extends PollOperationState<TResult>, TResult> {
+    // @deprecated
     cancelOperation(options?: {
```

Comment:
Could we create something like the storage migration example in the core-lro samples that shows, in general, how we recommend augmenting the poller with methods?

Reply:
Moving forward, I would like us to use `createPoller` for creating all new pollers. As for cancellation, I am proposing a pattern for it in the PR description that doesn't involve extending pollers at all.

Reply:
I will mark this as resolved but let me know if you disagree.